src/minisphere/galileo.c

```diff
@@ -409,10 +409,27 @@ shader_ref(shader_t* it)
 void
 shader_unref(shader_t* it)
 {
+	struct uniform *uniform;
+	iter_t iter;
```

Comment:
I wasn't 100% sure about this, but it seems like the prevailing style is to have the space after the `*`, not before.  Also, this is a bit of a weird style quirk, but `iter_t iter;` should be on its own line; I have this odd OCD thing where I refuse to declare variables inline but nonetheless insist of having "loop variables" set off from the rest.

Reply:
I changed this anyway to fit with the rest of the code - `*` with the type, and in columns. I forgot to add a blank line before iter - I can add that but I don't want to litter the commit history...

Reply:
It's fine, I typically do a squash merge anyway.